### PR TITLE
docs: fix broken link in results-manager template

### PR DESCRIPTION
**Pull request by mrrajamanickam-coveo**
## Proposed changes
* fix broken link in results-manager template

### CODE REVIEW

File: packages/ui/atomic/template/src/components/results-manager/results-manager.tsx.hbs [hunk 1]
```diff
@@ -6,7 +6,7 @@ import template{{inc @index}} from './template-{{inc @index}}.html';
 
 /**
  * Component used to manage results & result templates.
- * See https://docs.coveo.com/en/atomic/latest/usage/create-a-result-list/
+ * See https://docs.coveo.com/en/atomic/latest/usage/displaying-results/
```
Comment (mrrajamanickam-coveo) [inline packages/ui/atomic/template/src/components/results-manager/results-manager.tsx.hbs:9]: This looks like the potential replacement link - not sure if there is a better one out there - @jpmarceau ?

Comment (jpmarceau) [reply packages/ui/atomic/template/src/components/results-manager/results-manager.tsx.hbs:9]: I think that's good 👍

**Review by louis-bompart — APPROVED** [review]

**Review by y-lakhdar — APPROVED** [review]

**Review by jpmarceau — APPROVED** [review]

**Comment by mrrajamanickam-coveo** [comment]
Don't think the changes in this PR updating a doc string should cause the CI error
```
 › 3 snapshots failed.
Snapshot Summary
 › 3 snapshots failed from 1 test suite. Inspect your code changes or run `npm run jest:ci -- -u` to update them.
```

Proceeding to merge ..

**Comment by louis-bompart** [comment]
> Don't think the changes in this PR updating a doc string should cause the CI error
> ```
>  › 3 snapshots failed.
> Snapshot Summary
>  › 3 snapshots failed from 1 test suite. Inspect your code changes or run `npm run jest:ci -- -u` to update them.
> ```
> 
> Shall I proceed to merge @louis-bompart ?

Lgtm.
If you have time, stabilize snaps on main before. If not, no biggie.
Cause: release should update snaps, they don't.

**Comment by mrrajamanickam-coveo** [comment]
Not sure what updating snapshots means in the context of CI. Is it jest snapshots? Tried to follow the instruction "run `npm run jest:ci -- -u` to update them" from the test failure but `jest:ci` doesn't exist in pkg json - couldn't find any equivalent in pkg json either. 

Will merge this in the interest of getting the long open PR closed.

**Pull request merged**